masu/processor/expired_data_remover.py

```diff
@@ -15,16 +15,21 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 """Remove expired report data."""
+# pylint: skip-file
```

Comment:
why are we skipping pylint?

Reply:
We are doing this in a few other files due to this.  For some reason adding just the exception for duplicate-code is not working (maybe this https://github.com/PyCQA/pylint/issues/214):

```
from masu.external import (AMAZON_WEB_SERVICES,
                           AWS_LOCAL_SERVICE_PROVIDER,
                           OCP_LOCAL_SERVICE_PROVIDER,
                           OPENSHIFT_CONTAINER_PLATFORM)
from masu.external.date_accessor import DateAccessor (duplicate-code)
```

Reply:
We should include a comment next to the disable to explain why it's there and when it should be removed.